src/components/sharedComponents/TokenInput/index.tsx

```diff
@@ -47,12 +47,11 @@ export const CloseButton = styled(BaseCloseButton)`
 `
 
 interface TokenInputProps extends Omit<TokenSelectProps, 'onError'> {
-  singleToken?: boolean
-  token?: Token
   onAmountSet: (amount?: string) => void
   onError: (error?: string) => void
   thousandSeparator?: boolean
   title?: string
+  singleToken?: Token
```

Comment:
Doing this change causes it to lose one of the possible configurations.

1. If `singleToken == false` and no `token` is provided: any token is pre-selected, and the user can select any token from any chain. 
2. if `singleToken ==  false` and a `token` is provided: the select initializes with a pre-selected token, and the user can freely change chain and token. 
3. if `singleToken == true` and `token` is provided: the chain and token are fixed. The user cannot change anything related to the token.

4. I just realized we should do something similar with the chain, sometimes I would want to fix the chain so the user can't change it.

Reply:
oh, oh... 🤦 nevermind then. Let's leave it as it is

Reply:
@nicosampler I'd rolled back the changes and was looking at the chains implementation.

If you pass an array of 1 element to 'network' you fix the number of chains, and the user won't be able to change it.

Please, check if that works for you.

<img width="672" alt="image" src="https://github.com/user-attachments/assets/2fe1f5fd-8ebb-4cfd-872f-ac0a550b6841">